docs/manual/cluster-admin/how-to-add-and-remove-nodes.md

```diff
@@ -50,6 +50,8 @@ OpenPAI doesn't support changing master nodes, thus, only the solution of adding
       pai-worker: "true"
   ```
 
+- Make sure that you can access all nodes in the cluster using the settings in `<config-folder>/config.yaml`. If you use SSH key pairs to log in to nodes, please mount the folder `~/.ssh` on the dev box machine to `/root/.ssh` on the dev box docker container。
+
```

Comment:
Do we support ssh key pai log in during installation ? https://github.com/microsoft/pai/blob/master/docs/manual/cluster-admin/installation-guide.md#configyaml-example

Reply:
The user can leave the `password` blank and manually copy the SSH private key to nodes.